etc/kayobe/kolla/config/ironic/ironic-conductor.conf

```diff
@@ -0,0 +1,5 @@
+# Enable automated cleaning. To save time this uses ATA secure erase when
+# supported.
```

Comment:
nit: If we add any more options in here, this comment won't make much sense at the file level - move to above the option?

Reply:
Right you are sir. Fixed.